src/dart-2.md

```diff
@@ -55,6 +57,7 @@ It aims to help developers migrating from Dart 1.x to Dart 2.
 
 [angular-examples repos]: https://github.com/angular-examples
 [apiref]: {{site.dart_api}}/dev
+[assert statements]: /guides/language/language-tour#assert
 [build_runner web]: {{site.dev-webdev}}/tools/build_runner
 [creating library packages]: /guides/libraries/create-library-packages
 [Dart 2 changes]: /guides/language/spec#dart-2-changes
```

Comment:
Since we're already pointing folks there, how about making \[webdev dart2] refer to `{{site.dev-webdev}}`:

    [webdev dart2]: {{site.dev-webdev}}/dart-2